rotkehlchen/data/importer.py

```diff
@@ -602,27 +615,35 @@ def _import_cryptocom_associated_entries(self, data: Any, tx_kind: str) -> None:
                         )
                         self.db_ledger.add_ledger_action(action)
 
-    def import_cryptocom_csv(self, filepath: Path) -> Tuple[bool, str]:
+    def import_cryptocom_csv(self, filepath: Path, **kwargs: Any) -> Tuple[bool, str]:
         with open(filepath, 'r', encoding='utf-8-sig') as csvfile:
             data = csv.DictReader(csvfile)
             try:
                 #  Notice: Crypto.com csv export gathers all swapping entries (`lockup_swap_*`,
                 # `crypto_wallet_swap_*`, ...) into one entry named `dynamic_coin_swap_*`.
-                self._import_cryptocom_associated_entries(data, 'dynamic_coin_swap')
+                self._import_cryptocom_associated_entries(
+                    data,
+                    'dynamic_coin_swap',
+                    **kwargs,
+                )
```

Comment:
nitpick: use kwargs for method calls

```suggestion
                self._import_cryptocom_associated_entries(
                    data=data,
                    tx_kind='dynamic_coin_swap',
                    **kwargs,
                )
```

Why? What is the benefit? Glad you ask! So with kwargs (1) the interpreter can tell you even before it executes that there is a problem if a needed arg is missing. (2) you no longer rely on the order of the arguments.

So doing the mistake of giving `('dust_conversion', data)` will not be caught and will only hit you at runtime. But if you give them as kwargs order does not matter.